designer/server/src/common/components/tag-env/template.njk

```diff
@@ -0,0 +1,15 @@
+{%- from "govuk/components/tag/macro.njk" import govukTag -%}
+
+{%- switch env %}
+    {% case "test" %}
+      {% set classes = "govuk-tag--yellow" %}
+    {% case "production" %}
+      {% set classes = "govuk-tag--red" %}
+    {% default %}
+      {% set classes = "govuk-tag--grey" %}
+  {% endswitch -%}
+
+  {{ govukTag({
+    text: env | capitalize,
```

Comment:
If we use the `replace` filter we can keep `perf-test` (hyphenated) as the env name

```suggestion
    text: env | replace("-", " ") | capitalize,
```

Reply:
Yeah I've asked on Slack, doing that prevents use in URLs:

```js
new URL(`http://forms-runner/.${config.env}.cdp-int.defra.cloud`)
```